.storybook/main.js

```diff
@@ -0,0 +1,29 @@
+// .storybook/main.js
+const path = require('path');
```

Comment:
this is the new supported config 

https://storybook.js.org/docs/react/configure/overview